test/runtests.jl

```diff
@@ -41,8 +41,11 @@ end
         lcov = IOBuffer()
         # we only have a single file, but we want to test on the Vector of file results
         LCOV.write(lcov, FileCoverage[r])
-        fn = joinpath(datadir, "expected.info")
-        @test String(take!(lcov)) == read(fn, String)
+        expected = read(joinpath(datadir, "expected.info"), String)
+        if lowercase(get(ENV, "APPVEYOR", "false")) == "true"
```

Comment:
I'd just check `Sys.iswindows()` instead.

Reply:
Thanks, will do (I am a complete Julia newbie, so I have no idea about what's available; thank you very much for this feedback, it's very helpful)

Reply:
Very impressive work for a newbie! :smile:

Reply:
Oops, `Sys.iswindows()` is not defined in Julia 0.6. I guess I'll just go back to the old method, unless you have a clever workaround?

Reply:
`Compat.Sys.iswindows()`